provisioning/requirements.txt

```diff
@@ -21,3 +21,4 @@ geerlingguy.repo-remi
 geerlingguy.security
 geerlingguy.solr
 geerlingguy.varnish
+arknoll.selenium
```

Comment:
I've been adding roles in alpha order - can you stick your role at the top of the file instead? (OCD, I know... ;).